website_sale_product_brand/views/product_brand.xml

```diff
@@ -6,10 +6,10 @@
     <template id="product_brands" name="Brands">
         <t t-call="website.layout">
             <t t-set="additional_title">Product Brands</t>
-            <div id="wrap" t-attf-class="wrapper"> 
+            <div id="wrap" t-attf-class="wrapper">
```

Comment:
Indeed this is a bug, but I prefer not to touch that. I'm only touching this file to remove a menu, seems like the space stripper changed this, but I'd prefer not to change something that could have a meaning without investigating, if you don't mind.